api/lib/domain/services/recommendation-service.js

```diff
@@ -0,0 +1,67 @@
+const _ = require('lodash');
+
+const REACH_LEVEL_POINTS = 40;
+const DIFFICULTY_POINTS = 30;
+const PROGRESS_POINTS = 20;
+const FIRST_STEP_POINTS = 10;
+const DEFAULT_REACHED_LEVEL = 0;
+
+const FIRST_LEVEL = 1;
+
+function computeRecommendationScore(skills, maxSkillLevelInTargetProfile, knowledgeElements) {
+  const skill = _.maxBy(skills, 'difficulty');
+  const reachLevelInTube =  _getReachLevelInTube(knowledgeElements, skills);
+
+  const reachLevelScore = _computeReachedLevelScore(skill, reachLevelInTube);
+  const difficultyScore = _computeDifficultyScore(maxSkillLevelInTargetProfile, skill);
```

Comment:
En plus inverser les paramètres

```suggestion
  const difficultyScore = _computeDifficultyScore(maxSkillLevelInTargetProfile, skillOfMaxDifficulty);
```

Reply:
done